### PR TITLE
New package: typstyle-0.11.28

### PR DESCRIPTION
**Pull request by Bnyro**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

**Comment by Bnyro** [comment]
ref https://github.com/void-linux/void-packages/pull/50945#issuecomment-2186049022

File: srcpkgs/typstyle/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="libgit2-devel"
+depends="git"
+short_desc="Beautiful and reliable typst code formatter"
+maintainer="Bnyro <bnyro@tutanota.com>"
+license="Apache-2.0"
+homepage="https://github.com/Enter-tainer/typstyle/"
+changelog="https://raw.githubusercontent.com/Enter-tainer/typstyle/v${version}/CHANGELOG.md"
```
Comment (tranzystorekk) [inline srcpkgs/typstyle/template:13]:
```suggestion
changelog="https://raw.githubusercontent.com/Enter-tainer/typstyle/master/CHANGELOG.md"
```

File: srcpkgs/typstyle/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'typstyle'
+pkgname=typstyle
+version=0.11.27
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="libgit2-devel"
+depends="git"
```
Comment (tranzystorekk) [inline srcpkgs/typstyle/template:8]: This is only used in the build script to obtain git info that isn't there anyway in the distfile. We should patch out all usage of git from the build deps, scripts and source, or even better, convince upstream to feature-gate it or sth like that.

Comment (tranzystorekk) [inline srcpkgs/typstyle/template:8]:
```suggestion
checkdepends="git"
```

File: srcpkgs/typstyle/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'typstyle'
+pkgname=typstyle
+version=0.11.27
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
```
Comment (tranzystorekk) [inline srcpkgs/typstyle/template:6]: `wasm` feature seems only relevant when used as a library

```suggestion
hostmakedepends="pkg-config"
configure_args="--no-default-features --features=cli"
```

File: srcpkgs/typstyle/template (hunk 2 above)
Comment (Bnyro) [reply srcpkgs/typstyle/template:8]: Thank you for upstreaming the changes!

**Comment by tranzystorekk** [comment]
We probably won't miss much here by disabling tests on i686 altogether, the `reflexo` crate seems to be for javascript only.

**Pull request merged**